### ISSUE TITLE
"Argument list too long"

### ISSUE THREAD

**Issue by josephwb** [FePhyFoFum/phyx#36]
Involves programs that can take wildcards e.g. `pxcat`:
```bash
./pxcat -s *.phy -o foo.fa
-bash: ./pxcat: Argument list too long
```
The solution seems to be to pass a file which itself lists the input files. Unfortunately, this has the same problem (for some OSs e.g. Mac):
```bash
find *.phy > flist.txt
-bash: /usr/bin/find: Argument list too long
```
However, the following seems to work for both linux/mac:
```bash
for x in *.phy; do echo $x >> flist.txt; done
```
So `flist.txt` can be passed instead of the wildcard, and parsed when it needs to be.

**Comment by josephwb** [comment]
I've got this implemented in `pxcat`, just running it to make sure it works. Seems _really_ slow, though. I wonder if we might make it more efficient somehow.

**Comment by josephwb** [comment]
`pxcat` issue solved with dd0d08f84c3e94a7a871de961622ea66531c1204. Use this strategy with others.
